README.md

```diff
@@ -111,6 +111,89 @@ func quit(g *gocui.Gui, v *gocui.View) error {
 }
 ```
 
+### Example with Simulated Test Screen
+
+You can write simple tests for `gocui` which let you simulate keyboard and then validate the output drawn to the screen.
+
+1. Create an instance of `gui` with `OutputSimulator` set as the mode `g, err := NewGui(OutputSimulator, true)`
+1. Call `GetTestingScreen` to get a `testingScreen` instance. 
+1. On this you can use `SendKey` to simulate input and `GetViewContent` to evaluate what is drawn.
```

Comment:
Why is this all 1. ?

Reply:
Markdown will auto increment numbers for you. I've tended to be lazy and let it do it, that way if I add something in the middle I don't have to go fix them all up. 

Preview here shows this comes out as [1/2/3](https://github.com/awesome-gocui/gocui/blob/c54069f2067031a052baa502cd48de39fb53ce59/README.md)

Reply:
I'll fix this up now as you are right, it does make it more readable if you are looking at the markdown directly